### PR TITLE
1050 responsive header

### PR DESCRIPTION
**Pull request by jonathanbobel**

### CODE REVIEW

File: app/templates/components/components/header/template.njk [hunk 1]
```diff
@@ -44,61 +44,64 @@
...
-          {% endif %}
-        {% endfor %}
+            <li class="usa-nav__primary-item{{ ' is-current' if item.active }}">
+              <a class="usa-nav__link {{ ' usa-current' if item.active }}" href="{{ item.href }}" {% for attribute, value in
+                item.attributes %} {{attribute}}="{{value}}" {% endfor %}>
+                <span>{{ item.text }}</span>
+              </a>
+            </li>
+            {% endif %}
+          {% endfor %}
```
Comment (ccostino) [inline app/templates/components/components/header/template.njk:79]: Looks like there are a couple of extra empty spaces here that we can eliminate:

```suggestion
          {% endif %}
        {% endfor %}
```

**Review by ccostino — COMMENTED** [review]
Awesome, thanks @jonathanbobel!  I had just one minor suggestion.  Nice to see the leftover comments get removed, too!

**Comment by ccostino** [comment]
@jonathanbobel there might be one more tweak to make here.  The header works and functions great, but if you're on the landing page, you'll end up with an empty menu since there are no options:

Screenshot 2024-02-27 at 10 45 40 AM

Screenshot 2024-02-27 at 10 45 27 AM

Would it be possible to check if there is any menu content and if there isn't, don't show the menu button?

**Comment by jonathanbobel** [comment]
@ccostino should be good now

File: app/templates/components/components/header/template.njk [hunk 2]
```diff
@@ -44,61 +44,63 @@
...
+  <div class="usa-nav-container">
+    <div class="usa-navbar">
+      <div class="usa-logo display-flex flex-align-center flex-justify" id="-logo">
+        {# <div class="logo-img display-flex">
+          <span class="usa-sr-only">US Notify Logo</span>
+          <image src="{{ params.assetsPath | default('/static/images') }}/us-notify-color.png" alt="US Notify logo" xlink:href=""
+            class="usa-flag-logo margin-right-1" width="40" height="35"></image>
+        </div> #}
+        <em class="logo-text usa-logo__text">
+          <a href="/" title="notify.gov">notify.gov</a>
```
Comment (terrazoon) [inline app/templates/components/components/header/template.njk:56]: do we need capital N?

**Review by terrazoon — APPROVED** [review]
One little question but no blockers

File: app/templates/components/components/header/template.njk (hunk 2 above)
Comment (ccostino) [reply app/templates/components/components/header/template.njk:56]: That's a good catch, @terrazoon!

@maomeara63 or @stvnrlly, we have been using `Notify.gov` across the board, right?

Comment (jonathanbobel) [reply app/templates/components/components/header/template.njk]: got it

File: app/templates/components/components/header/template.njk [hunk 3]
```diff
@@ -44,61 +44,63 @@
...
+  <div class="usa-nav-container">
+    <div class="usa-navbar">
+      <div class="usa-logo display-flex flex-align-center flex-justify" id="-logo">
+        {# <div class="logo-img display-flex">
+          <span class="usa-sr-only">US Notify Logo</span>
+          <image src="{{ params.assetsPath | default('/static/images') }}/us-notify-color.png" alt="US Notify logo" xlink:href=""
+            class="usa-flag-logo margin-right-1" width="40" height="35"></image>
+        </div> #}
+        <em class="logo-text usa-logo__text">
+          <a href="/" title="notify.gov">Notify.gov</a>
```
Comment (ccostino) [inline app/templates/components/components/header/template.njk:56]: One more slight change!

```suggestion
          <a href="/" title="Notify.gov">Notify.gov</a>
```

**Review by ccostino — APPROVED** [review]
Thank you, @jonathanbobel!

**Pull request merged**